lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -3359,6 +3359,104 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
         rewriter.replaceOp(binder.op, inputSequence);
         return success();
       });
+  patterns.onOp(
+      "SequenceMap", 17,
+      [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+        Value inputSequence, additionalInput;
+        Torch::ListType resultType;
+        if (binder.tensorListOperandAtIndex(inputSequence, 0) ||
+            binder.tensorListResultType(resultType))
+          return failure();
+        // get additional input and check if it's a sequence
+        bool hasAdditionalInput = true, isAdditionalList = true;
```

Comment:
This operator takes a variadic number of inputs, meaning it could take like 5 arguments, each of which could be either a tensor or a sequence, correct?